folium/templates/fol_template.html

```diff
@@ -139,6 +139,11 @@
       {{ add_line }}
       {% endfor %}
 
+      {% for multiline, add_multiline in multilines %}

```

Comment:
@scari you are looping over `multilines` and not `multiline` {% for multiline, add_multiline in multilines %}
If you change it above you'll get it working properly.